packages/ui/src/components/molecules/MultiValuePickerButton/MultiValuePickerButton.tsx

```diff
@@ -50,28 +52,33 @@ const MultiValuePickerButton = ({
     <TouchableOpacity onPress={onPress} activeOpacity={0.9}>
       <Card style={[styles.container, style]}>
         <View style={styles.listItemContainer}>
-          {listItem &&
-            listItem.map((item, index) => (
-              <TouchableOpacity
-                onPress={() => onPressItem(item)}
-                disabled={readonly}
-                key={index}
-                style={[styles.cardItem, getItemColor(item.color)]}
-                activeOpacity={0.9}>
-                <Text
-                  style={styles.text}
-                  fontSize={14}
-                  numberOfLines={1}
-                  textColor={
-                    item.color.foreground || Colors.primaryColor.foreground
-                  }>
-                  {item.title}
+          {listItem && listItem.length > 0
+            ? listItem.map((item, index) => (
```

Comment:
All three placeholders should be display the same way. Currently, we have three different positions. Make them all follow the green rules : 

![image](https://github.com/axelor/axelor-mobile/assets/102581501/4f07c151-1d53-47fb-b2d2-598cfeb1bb09)